src/lib/components/lines-and-dots/workflow-details.svelte

```diff
@@ -23,27 +23,40 @@
   <div
     class="surface-primary grid w-full grid-flow-row grid-cols-1 gap-2 rounded-xl border-2 border-primary p-2 md:grid-cols-2 xl:grid-cols-3"
   >
-    <WorkflowDetail
-      title={translate('common.duration')}
-      content={elapsedTime}
-      class="order-1 text-sm"
-      icon="clock"
-    />
     <WorkflowDetail
       title={translate('common.start')}
+      tooltip={$relativeTime
```

Comment:
Probably my intense hatred for ternaries is clouding my judgement here, but maybe the tooltip should be an object with both time stamps or implemented in workflow detail so it always hits this codepath?